keyboards/studiokestra/line_tkl/info.json

```diff
@@ -0,0 +1,126 @@
+{
+    "manufacturer": "Studio Kestra",
+    "keyboard_name": "Line TKL",
+    "maintainer": "studiokestra",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F0", "F1", "F4", "F5", "F6", "F7", "B5", "D5", "D3"],
+        "rows": ["D0", "D1", "B0", "B7", "B3", "E6", "C7", "C6", "B4", "D7", "D6", "D4"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://minokeys.com/collections/line-friends",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x8704",
+        "vid": "0x7C10"
+    },
+    "indicators": {
+        "caps_lock": "B6",
+        "scroll_lock": "D2",
+        "on_state": 1
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
could a layout with 2u Backspace and 2.75u Right shift (`LAYOUT_tkl_ansi_tsangan`) be added?